### DISCUSSION TITLE
[Projects Beta] Charts: Archived items retroactively removed from X-axis Time e.g. Burn Up / CFD

### DISCUSSION THREAD

**Discussion by 700software** [community/community#17280] [Projects and Issues]
We were using this chart to keep track of how many issues were not in the Done status simply by clicking off each status we wanted hidden.

Now the chart is useless because we archived a bunch of done issues to improve loading time and reduce scrolling/filtering/reordering.

I wanted to see a down-turn of issue count as of the date it was archived, but not it's like the issue never existed. So our Time series chart is now longer useful.

Need Archived issues and Deleted issues to be counted I think? Only drop them off 'as of' the day they were archived or deleted.

See:
- https://github.com/github/feedback/discussions/10915#discussioncomment-2105943
- https://github.com/github/feedback/discussions/13356

**Comment by yuanchen-zhu** [comment]
I have the same issue. Burnup chart is useless since we archive DONE tasks from previous sprints. Would appreciate it if it's made a configurable option whether archived issues are included in the Burnup chart or not.

**Reply by chrmay** [reply]
Same situation here. Just realised that the diagrams we have in place are basically empty. Almost two years of issue tracking data gone 😭

**Comment by jeffrey-e** [comment]
Same challenge here. I wonder what the best practice is. I cant imagine that you are supposed to keep all your Done Issues as they will clog your views

**Comment by 700software** [comment]
I have recently realized that you eventually are forced to Archive things due to the 1,200 max items in Project. So 'just never Archive anything' is not only undesirable but also not a possible solution for larger projects.

**Comment by aeneasr** [comment]
For larger Enterprise organizations, even 50k item limit might not be enough, and it also makes it incredibly slow to use the board if you have more than a couple of thousand issues on there. The charts however are essentially just nice to look at but provide no real value, if they "retroactively" remove archived items from the stats.

**Comment by T3sT3ro** [comment]
Charts are unusable due to that issue.

**Comment by slinna** [comment]
Just wanted to let you know that there's a marketplace add-on called [Analytics & Reports](https://github.com/marketplace/analytics-reports) by Screenful, which allows tracking GitHub project data with [15 chart types](https://screenful.com/tour/charts) (including burndown and cumulative flow). The integration is built so that if you archive an item, it will remain in the stats so that you can track trends over longer time periods. If you're curious to learn more, here's the [guide for getting started](https://screenful.com/guide/github).
